multi-domain-ses/MultiDomainSES.cfc

```diff
@@ -17,7 +17,7 @@ Description :
 		<cfscript>
 		
 			// set the base URL according to domain or whatever strategy you like.
-			arguments.event.setSESBaseURL("http://" & cgi.http_host & "/index.cfm");
+			arguments.event.setSESBaseURL( ((cgi.https == "on")? "https://": "http://")  & cgi.http_host & "/index.cfm");
```

Comment:
Sana, why don't you instead of using the cgi scope, use the method `event.isSSL()` instead.